extensions/positron-python/package.json

```diff
@@ -689,7 +689,7 @@
                     "type": "string"
                 },
                 "python.locator": {
-                    "default": "js",
+                    "default": "native",
```

Comment:
Right now, the rust locator is considered experimental and is opt-in only for upstream. I've swapped it for testing purposes, but we'll want to decide what we want as the default.

I've played around with it locally, and it seems to be finding everything and is MUCH FASTER, eg, 1-3 seconds to find my 80+ envs.

Reply:
I've had the same experience. We could leave the default as `js` when we merge to `main` so that it's optional for the Feb release, and make it the default on `main` for devs to test during Feb in preparation for the Mar release?